projects/plugins/jetpack/modules/search/instant-search/components/search-result-expanded.jsx

```diff
@@ -28,51 +30,90 @@ export default function SearchResultExpanded( props ) {
+							dangerouslySetInnerHTML={ { __html: highlight.title } }
+						/>
+					</h3>
+
+					{ ! isMultiSite && (
+						<PathBreadcrumbs
+							className="jetpack-instant-search__search-result-expanded__path"
+							onClick={ props.onClick }
+							url={ `//${ fields[ 'permalink.url.raw' ] }` }
+						/>
+					) }
+
+					<div
+						className="jetpack-instant-search__search-result-expanded__content"
 						//eslint-disable-next-line react/no-danger
-						dangerouslySetInnerHTML={ { __html: highlight.title } }
+						dangerouslySetInnerHTML={ {
+							__html: highlight.content.join( ' ... ' ),
+						} }
 					/>
-				</h3>
-				<PathBreadcrumbs
-					className="jetpack-instant-search__search-result-expanded__path"
-					onClick={ props.onClick }
-					url={ `//${ fields[ 'permalink.url.raw' ] }` }
-				/>
-				<div
-					className="jetpack-instant-search__search-result-expanded__content"
-					//eslint-disable-next-line react/no-danger
-					dangerouslySetInnerHTML={ {
-						__html: highlight.content.join( ' ... ' ),
-					} }
-				/>
 
-				{ highlight.comments && <SearchResultComments comments={ highlight.comments } /> }
+					{ highlight.comments && <SearchResultComments comments={ highlight.comments } /> }
+				</div>
+				<a
+					className="jetpack-instant-search__search-result-expanded__image-link"
+					href={ `//${ fields[ 'permalink.url.raw' ] }` }
+					onClick={ props.onClick }
+				>
+					<div className="jetpack-instant-search__search-result-expanded__image-container">
+						{ firstImage ? (
+							// NOTE: Wouldn't it be amazing if we filled the container's background
+							//       with the primary color of the image?
+							<PhotonImage
+								alt={ highlight.title }
+								className="jetpack-instant-search__search-result-expanded__image"
+								isPhotonEnabled={ props.isPhotonEnabled }
+								src={ `//${ firstImage }` }
+							/>
+						) : null }
+					</div>
+				</a>
 			</div>
-			<a
-				className="jetpack-instant-search__search-result-expanded__image-link"
-				href={ `//${ fields[ 'permalink.url.raw' ] }` }
-				onClick={ props.onClick }
-			>
-				<div className="jetpack-instant-search__search-result-expanded__image-container">
-					{ firstImage ? (
-						// NOTE: Wouldn't it be amazing if we filled the container's background
-						//       with the primary color of the image?
+			{ isMultiSite && (
+				<ul className="jetpack-instant-search__search-result-expanded__footer">
+					<li>
 						<PhotonImage
-							alt={ highlight.title }
-							className="jetpack-instant-search__search-result-expanded__image"
-							isPhotonEnabled={ props.isPhotonEnabled }
-							src={ `//${ firstImage }` }
+							alt={ fields.blog_name }
+							className="jetpack-instant-search__search-result-expanded__footer-blog-image"
+							isPhotonEnabled={ false }
+							height={ 24 }
+							width={ 24 }
+							src={ fields.blog_icon_url }
+							lazyLoad={ false }
 						/>
```

Comment:
When performing a multi-site search, the site icon is always that of the current site:

<img width="1460" alt="Screen Shot 2021-06-22 at 1 54 45 PM" src="https://user-images.githubusercontent.com/4044428/122993140-325bcf00-d364-11eb-85c5-20187cdecc78.png">

Reply:
Good catch, should be fixed now.